### ISSUE TITLE
Wrong comment for STM32H7B0xx

### ISSUE THREAD

**Issue by carlo-dev-git** [STMicroelectronics/cmsis_device_h7#13]
Device **STM32H7B0xxQ** is not documented.
Line 74 in `stm32h7xx.h` should be changed from:
``` c
  /* #define STM32H7B0xx */   /*!< STM32H7B0ABIxQ, STM32H7B0IBTx, STM32H7B0RBTx, STM32H7B0VBTx, STM32H7B0ZBTx, STM32H7B0IBKxQ */
```
to:
``` c
  /* #define STM32H7B0xx */   /*!< STM32H7B0IBTx, STM32H7B0RBTx, STM32H7B0VBTx, STM32H7B0ZBTx Devices */
  /* #define STM32H7B0xxQ */  /*!< STM32H7B0ABIxQ, STM32H7B0IBKxQ Devices */
```

**Comment by RJMSTM** [comment]
ST Internal Reference: 180246

**Comment by TOUNSTM** [comment]
Fixed in 13ba9da797a30e1563a8795f974425fb26dd0457
